### ISSUE TITLE
`createTab incognito` creates a new window

### ISSUE THREAD

**Issue by A-Manning** [philc/vimium#3109]
Expected behaviour: `createTab incognito` should create a new tab in an existing window, if possible.

**Comment by gdh1995** [comment]
Any incognito tabs must in a incognito window since Chrome 52.

In fact, before 52 there're some tricks to move incognito tabs into normal windows, but this method was rarely used.

**Comment by A-Manning** [comment]
Yes, but those tabs should be part of an existing incognito window, if possible. Like how "Open link in incognito window" from the right-click menu works.
